### ISSUE TITLE
add info about how to check the status when upgrading on bluexp

### ISSUE THREAD

**Issue by Xinwei-Wang** [NetAppDocs/bluexp-cloud-volumes-ontap#210]
### Page URL

https://docs.netapp.com/us-en/bluexp-cloud-volumes-ontap/task-updating-ontap-cloud.html

### Page title

Upgrade Cloud Volumes ONTAP software

### Summary

Some customers are asking how to check the progress of the CVO upgrade on blueXP ,pleas add the info under this page.
https://docs.netapp.com/us-en/bluexp-cloud-volumes-ontap/task-updating-ontap-cloud.html#upgrade-from-bluexp-notifications

we can check the status from the timeline.

also if possible please also add more info about if the upgrade fails since reverting is not supported. 

Reverting or downgrading
Reverting or downgrading Cloud Volumes ONTAP to a previous release is not supported.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-driley** [comment]
Hi, 

Thank you so much for your feedback. We will review this issue and revise the documentation appropriately. We will keep you posted when the updated documentation will be available. Please let us know if you have any other feedback. 

Thanks again, 

Don Riley

**Comment by netapp-rlithman** [comment]
I drafted the following additional step 6 to the section Upgrade from BlueXP notifications. 

6. To check the status of the upgrade, click the Settings icon and select Timeline. 

I looked for the corresponding action in the Timeline for an upgrade but I couldn't find one unless it's called "Upgrade Service Manager". What is the "action" in the Timeline that users should look for when they want to check the status of the upgrade? 

For information on what to do when an upgrade fails, do we have any KB articles or other Support information that customers can refer to?
